src/platform/pdf/templates/veteran_status_new.js

```diff
@@ -0,0 +1,305 @@
+  // VA logo
+  if (data.details.image) {
+    const logoGraphic = await fetchImage(data.details.image.url);
+
+    const logoImage = doc.image(
+      logoGraphic,
+      doc.page.margins.left,
+      doc.page.margins.top,
+      { width: 155, alt: data.details.image.title },
+    );
+
+    const logo = doc.struct('Figure', { alt: data.details.image.title }, [
+      () => logoImage,
+    ]);
+
+    wrapper.add(logo);
+  }
+
+  doc.moveDown(0.5);
+
+  // Heading
+  const heading = createHeading(
+    doc,
+    'H1',
+    config,
+    'Proof of Veteran status card',
+    {
+      x: doc.page.margins.left,
+      paragraphGap: 10,
+    },
+  );
+  wrapper.add(heading);
+
+  // description
+  wrapper.add(
+    doc.struct('P', () => {
+      doc
+        .font(config.paragraph.font)
+        .fontSize(config.paragraph.size)
+        .text(
+          'You can use your Veteran status card to get discounts at stores, businesses, and restaurants.',
+          doc.page.margins.left,
+          doc.y,
+          {
+            lineGap: 4,
+            width: 450,
+          },
+        );
+    }),
+  );
+
+  // veteran status card
+  // Add content synchronously to ensure that reading order
+  // is left intact for screen reader users.
+
+  doc.moveDown(1);
+  const cardSection = doc.struct('Sect');
+  wrapper.add(cardSection);
+
+  const cardYPosition = doc.y;
+  const cardXPosition = doc.x;
+  const cardWidth = 252; // roughly results in a 2 x 3.5 inch rectangle
+  const cardHeight = 144;
+  const cardPadding = 12;
+
+  // Add a dotted line to indicate where the card should be cut out
+  const cardWrapper = doc.struct('Artifact', () => {
+    doc
+      .lineWidth(0.5)
+      .roundedRect(doc.page.margins.left, doc.y, cardWidth, cardHeight, 8)
+      .dash(3.5, { space: 3.5 })
+      .stroke();
+  });
+
+  cardSection.add(cardWrapper);
+
+  // Card title
+  doc.moveDown(0.5);
+  const cardHeading = doc.struct('H2', () => {
+    doc
+      .font(config.headings.H2.font)
+      .fontSize(config.headings.H2.size)
+      .text(
+        'Proof of Veteran Status',
+        doc.page.margins.left + cardPadding,
+        doc.y,
+      );
+  });
+  cardSection.add(cardHeading);
+
+  // VA seal
+  if (data.details.seal) {
+    const sealGraphic = await fetchImage(data.details.seal.url);
+    const sealWidth = 40;
+    const sealImage = doc.image(
+      sealGraphic,
+      cardXPosition + cardWidth - cardPadding - sealWidth, // positioned relative to top-right corner
+      cardYPosition + cardPadding,
+      { width: sealWidth, alt: data.details.image.title },
+    );
+    const seal = doc.struct('Figure', { alt: data.details.seal.title }, [
+      () => sealImage,
+    ]);
+    cardSection.add(seal);
+  }
+
+  // First column of info items
+  doc.moveDown(0.25);
+  const infoItems = [
+    {
+      heading: 'Name',
+      content: data.details.fullName,
+    },
+    {
+      heading: 'Latest period of service',
+      content: `${data.details.latestService}`,
```

Comment:
This 'new' version of the PDF template uses the `latestService` that it is passed directly instead of performing its own algorithm to find it given a service history.